document_guides/requirement_guide.md

```diff
@@ -1,19 +1,80 @@
 # Requirement Guide
 
-In the `documents` folder of each project, there is a `reqirement.md` specifying the business requirment of the project.
+In the `docs` folder of each project, there is requirement document specifying the business requirment of the project.
+For different usage, name of document should be different:
 
-the requirement should contains the fellow content:
+* `requirement.md`
 
-### The purpose of this project
-we should describe the purpose of this project, why we start this project.
- 
-### The output of this project
-this may be a jar file, jar file, docker image etc. 
+   if this micro service is single usage, one document is enough.
 
-### The king of service provided
-Rest API, grpc api, or web content etc.
+* `admin_requirement.md`
 
-### The detail of the service or content that this project provided
+  requirement document for admin-web.
 
+* `customer_requirement.md`
 
+  requrement document for customer-web.
 
+## 1. Document Structure
+
+All requirement document is `markdown` file, and the structure should be similar, here is schema:
```

Comment:
All requirement documents should be  `markdown` file, and the structure should be similar, here is schema: